h2o-docs/src/product/parameters.rst

```diff
@@ -100,6 +100,7 @@ This Appendix provides detailed descriptions of parameters that can be specified
    data-science/algo-params/offset_column
    data-science/algo-params/pca_impl
    data-science/algo-params/pca_method
+   data-science/algo-params/plug_values
```

Comment:
Delete this change since we are not adding this file. The docs will produce an error during the build.